schema/pet-insurance.json

```diff
@@ -152,17 +201,649 @@
                 "petType": {
                     "type": "string",
                     "description": "",
-                    "example": "dog"
+                    "example": "dog",
+                    "enum": [
+                        "dog",
+                        "cat"
+                    ]
                 },
                 "petGender": {
                     "type": "string",
                     "description": "",
                     "example": "Male"
                 },
+                "breedType": {
+                    "type": "string",
+                    "description": "",
+                    "example": "Male"
+                },
                 "breedName": {
                     "type": "string",
                     "description": "Your pet breed",
-                    "example": "Afgan"
+                    "example": "Afgan",
+                    "enum": [
+                        "Dogs:",

```

Comment:
Because that's what I understood from specifying in the enum breed name categories for dogs and cats. Can you give me an example of how this enum should look with dogs and cats breed names?

Reply:
```
  "enum": [
    "catBreed1",
    "catBreed2",

    "dogBreed1",
    "dogBreed2",
    "dogBreed3"
]
```

Reply:
I see. I'll change.